app/[locale]/[[...path]]/page.tsx

```diff
@@ -9,7 +9,11 @@ import WithLayout from '@/components/withLayout';
 import { ENABLE_STATIC_EXPORT, VERCEL_REVALIDATE } from '@/next.constants.mjs';
 import { PAGE_VIEWPORT, DYNAMIC_ROUTES } from '@/next.dynamic.constants.mjs';
 import { dynamicRouter } from '@/next.dynamic.mjs';
-import { availableLocaleCodes, defaultLocale } from '@/next.locales.mjs';
+import {
+  allLocales,
```

Comment:
This variable should probably be named `allLocaleCodes`

Reply:
But I'm also afraid of people mistaking or not understanding the difference between availableLocaleCodes and allLocaleCodes, which means probably availableLocaleCodes should be renamed to activeLocaleCodes